### PR TITLE
Update help documentation for 3.0.21-beta21 release

### PR DESCRIPTION
**Pull request by alerickson**
# PR Summary
Update help documentation for all cmdlets for 3.0.21-beta21 release.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Documentation needed]()
        - [ ] Issue filed: 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed]()
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary.

### CODE REVIEW

File: help/Publish-PSResource.md [hunk 1]
```diff
@@ -109,40 +108,8 @@ Type: System.String
...
-Parameter Sets: (All)
-Aliases:
-
-Required: True (None) False (All)
-Position: Named
+Required: True
+Position: 0
 Default value: None
 Accept pipeline input: False
 Accept wildcard characters: False
```
Comment (sdwheeler) [inline help/Publish-PSResource.md:115]: Were the **Proxy?** parameters removed? If so the syntax is wrong.

File: help/Save-PSResource.md [hunk 2]
```diff
@@ -302,14 +300,14 @@ minimum inclusive range, use `[1.0.0.0, ]` as the version range.
...
+Parameter Sets: IncludeXmlParameterSet, AsNupkgParameterSet
 Aliases:
 
 Required: False
 Position: Named
 Default value: None
-Accept pipeline input: False
-Accept wildcard characters: True
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
```
Comment (sdwheeler) [inline help/Save-PSResource.md]: This should be `True`. Was the cmdlet fixed?

**Review by sdwheeler — CHANGES_REQUESTED** [review]
See my comments. Also, docs are live in the docs repo now. So, after this PR, I will copy the changes to the docs repo and we need to delete the files here.

File: help/Publish-PSResource.md (hunk 1 above)
Comment (alerickson) [reply help/Publish-PSResource.md:115]: `Proxy` and `ProxyCredential` were previously duplicated, we just got rid of one set.

File: help/Save-PSResource.md (hunk 2 above)
Comment (alerickson) [reply help/Save-PSResource.md]: This should be true, we need to add an attribute to the cmdlet.

**Review by anamnavi — APPROVED** [review]

File: help/Publish-PSResource.md (hunk 1 above)
Comment (sdwheeler) [reply help/Publish-PSResource.md:115]: ok

**Review by sdwheeler — APPROVED** [review]

**Pull request merged**